docs/_tutorials/tcp-routes-mqtt.md

```diff
@@ -0,0 +1,406 @@
+---
+layout: tutorials
+title: TCP Routes for IoT - MQTT Java Application
+summary: A Simple Java Application using MQTT showing how to directly consume the a SERVICE KEY for a Solace Messaging service when using TCP Routes.
+icon: empty-logo.png
+---
+
+## Overview
+
+This tutorial is part of a series of tutorials which aims to introduce users to Solace Messaging in Pivotal Cloud Foundry. Solace Messaging in Pivotal Cloud Foundry is delivered as a Tile on the [Pivotal Network](https://network.pivotal.io/){:target="_blank"}. You can see the [Solace Messaging for Pivotal Cloud Foundry Documentation](http://docs.pivotal.io/solace-messaging/){:target="_blank"} for full details.
+
+This tutorial will introduce you to Solace Messaging for Pivotal Cloud Foundry by creating a Java application which connects to a Solace Messaging service with enabled TCP Routes as trivial example of an IoT use case.
+TCP Routes is a feature that allows a Solace Messaging service hosted in a Pivotal Cloud Foundry deployment to become accessible outside the PCF domain.
+
+
+![overview]({{ site.baseurl }}/images/tcp-routes-architecture.png){: .center-image}
+
+## Goals
+
+The goal of this tutorial is to demonstrate extracting the information from the application's Cloud Foundry provided Service Keys and connect to the Solace Messaging service instance.  This tutorial will show you:
+

```

Comment:
At this point can you outline what the goals are in terms of where the application should run. I.e. for tcp-routes it is applicable when run outside of a PCF environment I believe?